tests/ConsumptionSaving/test_ConsAggShockModel.py

```diff
@@ -59,7 +58,7 @@ def test_macro(self):
         self.economy.solve()  # Solve for the general equilibrium of the economy
 
         self.economy.AFunc = self.economy.dynamics.AFunc
-        self.assertAlmostEqual(self.economy.AFunc.slope, 1.16332, places=HARK_PRECISION)
+        self.assertAlmostEqual(self.economy.AFunc.slope, 1.13859, places=HARK_PRECISION)
 
```

Comment:
Now that seeds are properly set, could this commented-out simulation test be re-enabled?